letsencrypt/auth_handler.py

```diff
@@ -103,6 +104,28 @@ def _choose_challenges(self, domains):
             self.dv_c.extend(dom_dv_c)
             self.cont_c.extend(dom_cont_c)
 
+    def _simple_verify(self, achall, response):
+        """Self-verifies a response with the corresponding challenge"""
+        return_value = True
+
+        if isinstance(response, challenges.HTTP01Response):
+            return_value = response.simple_verify(
```

Comment:
`return response.simple_verify` here and below, `return True` in the `else` branch